### PR TITLE
Milestone 2: Portfolio Project

### PR DESCRIPTION
**Pull request by DannieMo**
Milestone 2: Portfolio Website

This second milestone of my portfolio website focuses on the sections detailing the list of my portfolio projects, e.g Multi-Purpose Stories.
Click the link in my README.md file to view.😃

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,5 +71,87 @@ <h2>Software Developer</h2>
 
        </div>
    </section>
+
```
Comment (o-rashad) [inline index.html:74]: Good work, following HTML best practices using HTML semantics 😎 
which will defiantly enhance your SEO ranking. 💹 

_You could add also `<aside>` , `<article>` and `<summery>` to your collection ➕_

File: index.html [hunk 2]
```diff
@@ -71,5 +71,87 @@ <h2>Software Developer</h2>
```
Comment (o-rashad) [inline index.html:71]: Congrats on the hard work you put into that project ^^!
Your PR looks so professional with a descriptive message. 💯 😄 

_Area to improve:_
_- Add more one-line descriptive commits._

**Review by o-rashad — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

Hi @DannieMo  👋😄, I'm Omar and I will be your reviewer for this project 😊

Good job so far!👌😃

There are some issues that you still need to work on to go to the next project but you are **almost there!** 🏁🏁🏁


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_NONE_

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -71,5 +71,87 @@ <h2>Software Developer</h2>
...
+                            standard dummy text ever since the 1500s, when an unknown 
+                            printer took a standard dummy text.
+                        </p>
+                        <div class="skills">
+                            <p>css</p>
+                            <p>html</p>
+                            <p>bootstrap</p>
+                            <p>Ruby</p>
+                        </div>
+                        <button type="button">See Project</button>
```
Comment (o-rashad) [inline index.html:95]: Well-made mocked UI design from your Figma Template ^^! 
_Impressive_

[Required] There is another small requirement that needs to be added 🔽 

fa8b3dff-60a9-4b8b-9d44-6be30387cd53

File: index.html [hunk 4]
```diff
@@ -71,5 +71,87 @@ <h2>Software Developer</h2>
...
+                         <p>A daily selection of privately personalized reads; 
+                             no accounts or sign-ups required. has been the industry's 
+                             standard dummy text ever since the 1500s, when an unknown 
+                             printer took a standard dummy text.
+                         </p>
+                         <div class="skills">
+                             <p>css</p>
+                             <p>html</p>
+                             <p>bootstrap</p>
+                             <p>Ruby</p>
```
Comment (od-c0d3r) [inline index.html:110]: [Required] Each project technology needs to be nested in the `<ul>` tag and each technology with `<li>` tag. 👍🏼

**Review by Emmyn5600 — APPROVED** [review]
## Status: Approved :heavy_check_mark: :muscle::clap:

Hi @DannieMo 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it :+1: .
Congratulations! :tada:

### Optional suggestions
- [ ] You need to make more commits or to commit oftentimes instead of pushing your changes at once.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by o-rashad** [comment]
Well done @DannieMo  😎

**Pull request merged**